src/Abstractions/TaskOrchestrationContext.cs

```diff
@@ -108,15 +108,30 @@ public abstract class TaskOrchestrationContext
     /// <see cref="TaskFailedException.FailureDetails"/> property.
     /// </exception>
     public virtual Task CallActivityAsync(TaskName name, object? input = null, TaskOptions? options = null)
-    {
-        return this.CallActivityAsync<object>(name, input, options);
-    }
+        => this.CallActivityAsync<object>(name, input, options);
+
+    /// <returns>
+    /// A task that completes when the activity completes or fails. The result of the task is the activity's return value.
+    /// </returns>
+    /// <inheritdoc cref="CallActivityAsync(TaskName, object?, TaskOptions?)"/>
+    public virtual Task CallActivityAsync(TaskName name, TaskOptions options)
+        => this.CallActivityAsync(name, null, options);
```

Comment:
Should we really inherit the docs from the `CallActivityAsync` with the input parameter?

Reply:
Same comment for the changes in CallSubOrchestrator

Reply:
To be honest, I don't really know the answer to that. @cgillum suggested I do it on another PR so I did 🤷

Reply:
Well, even if this were off, it's a non-blocking concern